versions/3.0.md

```diff
@@ -1688,14 +1858,14 @@ Field Pattern | Type | Description
 The link SHALL reference a single Link Object, or a JSON Reference to a single link object.
 
 
-### Link Object
+### <a name="#linkObject"></a>Link Object
 The `Link Object` is responsible for defining a possible operation based on a single response.
 
 Field Name  |  Type  | Description
 ---|:---:|---
 href        | `string`    | a relative or absolute URL to a linked resource. This field is mutually exclusive with the `operationId` field.
 operationId | `string` | the name of an _existing_, resolvable OAS operation, as defined with a unique `operationId`.  This field is mutually exclusive with the `href` field.  Relative `href` values MAY be used to locate an existing Operation Object in the OAS.
-parameters  | [Link Parameters Object](#linkParameters) | an Object representing parameters to pass to an operation as specified with `operationId` or identified via `href`.
+parameters  | [Link Parameters Object](#linkParameters) | an object representing parameters to pass to an operation as specified with `operationId` or identified via `href`.
```

Comment:
Since you've done all this work, mind fixing the anchor name to `#linkParametersObject` in all places?